### PR TITLE
Updated login error message

### LINKED ISSUE #1727: Update error message and make it more informative
### Problem

When we try to login with the wrong password, we get the following error message.
image

### Solution

It should be something more informative.
`Username doesn't exist` or `Incorrect password`.

### PR DESCRIPTION
**Pull request by Silver-IT**
Summary of Changes

- Closes #1727

### CODE REVIEW

File: frontend/controller/actions/identity.js [hunk 1]
```diff
@@ -357,17 +357,19 @@ export default (sbp('sbp/selectors/register', {
...
+      try {
         const salt = await sbp('gi.actions/identity/retrieveSalt', username, password)
         const IEK = await deriveKeyFromPassword(CURVE25519XSALSA20POLY1305, password, salt)
-
-        return [{ key: IEK, transient: true }]
-      })()
-      : []
+        transientSecretKeys.push({ key: IEK, transient: true })
+      } catch (e) {
+        throw new GIErrorUIRuntimeError(L('Incorrect username or password'))
```
Comment (taoeffect) [inline frontend/controller/actions/identity.js:364]: Please log the error here using `console.error`, e.g.:

```js
console.error('caught error calling retrieveSalt:', e)
```

**Review by taoeffect — CHANGES_REQUESTED** [review]
Nicely done, but some merge conflicts that need fixing + a request

**Review by taoeffect — APPROVED** [review]
Nicely done @Silver-IT! 👏

**Review by taoeffect — COMMENTED** [review]
~~Note: I had to undo this PR merge because it broke the flow after creating a group (no longer redirected to dashboard)~~

See: https://github.com/okTurtles/group-income/issues/1720#issuecomment-1773278280

**Comment by Silver-IT** [comment]
I am on sure why this PR is related to [this error](https://github.com/okTurtles/group-income/issues/1720#issuecomment-1773278280), and why should be undone. I haven't made any changes except removing IIFE and adding try/catch.

I think that error is related to any other stuff. Maybe related to login process.

**Pull request merged**